test/test_unit_keystore_credentials_provider.py

```diff
@@ -30,16 +35,66 @@ def test_load_credentials_do_not_exist_raises_credential_retrieval_exception(sel
             credential_provider = KeystoreCredentialsProvider()
             credential_provider.load('https://someurl')
 
+    @patch.object(KeystoreCredentialsProvider, 'is_exists', return_value=True)
+    @patch.object(KeystoreAdapter, "get_password", return_value="password")
+    def test_load_credentials_calls_get_password(self, mock_store_adapter, mock_cred_provider):
+        credential_provider = KeystoreCredentialsProvider()
+        credentials_data = credential_provider.load('https://someurl')
+        calls = [call('https://someurl', 'machine'), call('https://someurl', 'login'),
+                 call('https://someurl', 'password')]
+        mock_store_adapter.assert_has_calls(calls)
+        self.assertEquals("password", credentials_data.machine)
+        self.assertEquals("password", credentials_data.login)
+        self.assertEquals("password", credentials_data.password)
```

Comment:
the value should be different for machine and login

Reply:
Done